docs/brand/copywriting/naming.html

```diff
@@ -70,55 +66,31 @@
         <li>Don’t use ‘Private Q&A’ when referring to our product suite for marketing and promotional efforts outside of our core stackoverflow.com audience.</li>
     </ul>
 
-    {% header "h2", "Collectives on Stack Overflow" %}
+    {% header "h3", "Knowledge Reuse" %}
 
-    {% header "h3", "Capitalization" %}
+    <p class="stacks-copy">“Knowledge reuse” is a term we use frequently in reference to Stack Overflow for Teams as something it facilitates. It can be lowercase the majority of the time, except when refering to our <a href="https://stackoverflowteams.com/c/stack-overflow/questions/12132" title="Employees only have access to this">{% icon "Lock", "va-middle", "14" %} propritary metric</a>, in which case use “Knowledge Reuse”.</p>
```

Comment:
Just a couple misspellings
```suggestion
    <p class="stacks-copy">“Knowledge reuse” is a term we use frequently in reference to Stack Overflow for Teams as something it facilitates. It can be lowercase the majority of the time, except when referring to our <a href="https://stackoverflowteams.com/c/stack-overflow/questions/12132" title="Employees only have access to this">{% icon "Lock", "va-middle", "14" %} proprietary metric</a>, in which case use “Knowledge Reuse”.</p>
```